platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -1463,6 +1468,27 @@ MGL_EXPORT
  */
 @property (nonatomic, null_resettable) NSExpression *textPitchAlignment;
 
+/**
+ Radial offset of text, in the direction of the symbol's anchor. Useful in
+ combination with `textVariableAnchor`, which doesn't support the
+ two-dimensional `textOffset`.
+ 
+ This property is measured in ems.
```

Comment:
ems in relation to what? is it ONE_EM = 24.0? Other offsets are CGVector’s wrapped in NSValue’s, we should probably mention the underlying type in this documentation.

Reply:
yeah, it's 24.0

Reply:
we've few properties already in `ems` (e.g. https://github.com/mapbox/mapbox-gl-native/blob/master/platform/darwin/src/MGLSymbolStyleLayer.h#L1321) the underlying type info will appear when we update gl-gs version to one containing https://github.com/mapbox/mapbox-gl-js/commit/c4a22bf78e661114ac2da0481c6a4e0eee57122f